### ISSUE TITLE
Modal doesn't show on Windows OS with fade transition

### ISSUE THREAD

**Issue by jacobg** [bootstrap-vue-next/bootstrap-vue-next#2203]
### Describe the bug

Modal with default fade transition opens fine on Mac, but not on Windows OS. But if the no-fade prop is added to the modal, then it displays fine on Windows. It seems there's a Windows issue with transitions or animations?

This is not a recent bug. It existed at least as far back as 0.24.8.

### Reproduction

https://stackblitz.com/edit/github-jwsdlr-faxztu?file=src%2Fcomponents%2FComp.vue,src%2Fcomponents%2FMyModal.vue,src%2FApp.vue

### Used Package Manager

pnpm

### Validations

- [X] Have tested with the latest version. This is still alpha version and sometime things change rapidly.
- [X] Follow our [Code of Conduct](https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by xvaara** [comment]
tested, works on my windows, chrome and edge. Which browser are you using? Have you tried disabling adblockers?

**Comment by jacobg** [comment]
Chrome 128 on Windows 11 using a lambdatest vm, plus many of my customers experienced this regression which I observed using a screen share on their computers.

**Comment by VividLemon** [comment]
Atm you seem to be the best candidate for fixing and testing the issue. I dont use windows and cant fix what i cant see

**Comment by jacobg** [comment]
I worked around for it now by setting `no-fade`. I'll see if I can look at it at some point, but I'm not sure when that will be. I don't use Windows either, so never noticed this issue until I launched live and customers on Windows started reporting it. Then I tested on lambdatest vm and confirmed the issue.

**Comment by ptdev** [comment]
Hello,

I just tried this issue on my end and can confirm this is happening, but is apparently by design.

There are these particular CSS rules on bootstrap (this one is just for the fade rule, but there are others)

```
@media (prefers-reduced-motion:reduce) {
    .fade {
        transition: none
    }
}
```

Which essentially disable animations and effects if the user has the prefers-reduced-motion preference enabled.

And on Windows, if you have animations disabled on the OS (see screenshot below) then Chrome (and maybe other browsers) will enable the prefers-reduced-motion property and therefore disable animations.

![image](https://github.com/user-attachments/assets/9c81a708-0568-4e06-a6e9-09335719c8d7)

Enabling OS animations make the BS modal fade work again.

Some more info on this article:

https://www.tpgi.com/short-note-on-prefers-reduced-motion-and-puzzled-windows-users/

Hope it helps, cheers 👍

**Comment by xvaara** [comment]
I've previously looked at the bmodal transition and it's not good. 

To my understanding `enterToClass: 'show'` is added ant the beginning of transition, and it's removed at the after enter, so we add it back with isActive, which is triggeded in after-enter. Now that reduced motion is on and modelValue is true, they happen at the same tick, so transition removes the show class.

I have started to fix this in: #1821 but got a little stuck not getting the transition to work correctly, so I gave up. Now at least I have some motivation to get it to work.
